install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -853,6 +853,13 @@ gatewayAPI:
   # -- Control how traffic from external sources is routed to the LoadBalancer Kubernetes Service for all Cilium GatewayAPI Gateway instances. Valid values are "Cluster" and "Local".
   # ref: https://kubernetes.io/docs/reference/networking/virtual-ips/#external-traffic-policy
   externalTrafficPolicy: Cluster
+
+  gwClass:
```

Comment:
probably just use gatewayClass in full form to align with other existing field (e.g. gatewayAPI)

Reply:
Changed.